lib/provider/camunda/parts/implementation/ProcessVariables.js

```diff
@@ -86,7 +86,7 @@ module.exports = function(element, translate) {
       byScope = groupByScope(withNames);
 
   // (1) tab description entry
-  var description = entryFieldDescription(translate('List of variables written in this diagram.'));
+  var description = entryFieldDescription(translate('List of variables written in this diagram.'), translate);
```

Comment:
We should rebase this to master so we don't lose the wording changes, cf. https://github.com/bpmn-io/bpmn-js-properties-panel/commit/f32e424b5aea4ab2dd77f525c555c2959b4207da

Reply:
Done.